skore/src/skore/_sklearn/_plot/metrics/confusion_matrix.py

```diff
@@ -249,42 +184,50 @@ def _compute_data_for_display(
         cm = sklearn_confusion_matrix(
             y_true=y_true_values,
             y_pred=y_pred_values,
-            normalize=normalize,
```

Comment:
small details but could you write explicitly `normalize=None` to show that we force to not have any normalization at this stage.